libs/clients/national-registry/v1/src/lib/dto/getViewISLBorninMin.dto.ts

```diff
@@ -0,0 +1,119 @@
+export interface GetViewISLBorninMinDto {
+  message: string
+  table: Table
+  success: boolean
+}
+
+interface Table {
+  schema: Schema
+  diffgram: Diffgram
+}
+
+interface Diffgram {
+  DocumentElement: DocumentElement
+}
+
+interface DocumentElement {
+  ISLBorninMin: ISLBorninMin[]
+}
+
+export interface ISLBorninMin {
```

Comment:
aren't most of these nullable?

Reply:
whoops, only changed that in one place. good catch.